cpu/kinetis_common/include/periph_cpu.h

```diff
@@ -309,10 +309,18 @@ enum {
 /** @} */
 
 /**
+ * @brief UART hardware module types
+ */
+enum {
+    KINETIS_UART,   /**< Kinetis UART module type */
+    KINETIS_LPUART, /**< Kinetis Low-power UART (LPUART) module type */
+};
+
+/**
  * @brief UART module configuration options
  */
 typedef struct {
-    UART_Type *dev;               /**< Pointer to module hardware registers */
+    void *dev;                    /**< Pointer to module hardware registers */
```

Comment:
The macros are used as a way to eliminate the dispatcher function if a board only supports one or the other type of hardware.